resources/migrations/000_migrations.yaml

```diff
@@ -10970,6 +10970,31 @@ databaseChangeLog:
                   name: collection_id
             indexName: idx_collection_bookmark_collection_id
 
+  - changeSet:
+      id: v43.00-042
+      author: noahmoss
+      comment: >-
+        Added 0.43.0. Grant download permissions for existing Databases to 'All Users' permissions group
+      changes:
+        - sql:
+            sql: >-
+              INSERT INTO permissions (object, group_id)
+              SELECT db.object, all_users.id AS group_id
+              FROM (
+                SELECT concat('/download/db/', id, '/') AS object
+                FROM metabase_database
+              ) db
+              LEFT JOIN (
+                SELECT id
+                FROM permissions_group
+                WHERE name = 'All Users'
+              ) all_users
+                ON true
+              LEFT JOIN permissions p
+                     ON p.group_id = all_users.id
+                    AND db.object = p.object
+              WHERE p.object IS NULL;
```

Comment:
Do we want to grant download perms for all databases, or just the ones that `All Users` has some sort of permissions for? I guess we do actually want to do it for all Databases since if they have Collection perms but no ad-hoc data perms we'd still want them to be able to download those rows?

Reply:
Yeah, exactly. Even if you don't have ad-hoc query perms you should still be able to download data that you can view.